src/components/Stakeholders/StakeholderDetail/List.js

```diff
@@ -0,0 +1,65 @@
+import React, { useState, useContext } from 'react';
+import { makeStyles } from '@material-ui/core/styles';
+import { Paper, IconButton, Grid, ListItem } from '@material-ui/core';
+import UnlinkIcon from '@material-ui/icons/LinkOff';
+import LinkIcon from '@material-ui/icons/Link';
+import { StakeholdersContext } from '../../../context/StakeholdersContext';
+import ParentChildListItem from './ParentChildListItem';
+
+const useStyles = makeStyles({
+  listItem: {
+    backgroundColor: 'white',
+    marginBottom: 4,
+    borderRadius: 4,
+  },
+});
+
+function StakeholderList({ id, data, type, linked }) {
+  const classes = useStyles();
+  const [isLinked, setIsLinked] = useState(linked);
+  const { deleteStakeholder } = useContext(StakeholdersContext);
+
+  const handleChange = (e, data) => {
+    setIsLinked(!isLinked);
```

Comment:
I don't expect linked to be a state in this way. The `unlink` icon signifies an action to unlink the item from the focussed stakeholder (i.e., delete the relationship). It will always be an `unlink` icon.